src/java.base/share/classes/java/net/URLClassLoader.java

```diff
@@ -124,7 +124,7 @@ public URLClassLoader(URL[] urls, @Nullable ClassLoader parent) {
         this.ucp = new URLClassPath(urls, acc);
     }
 
-    URLClassLoader(String name, URL[] urls, ClassLoader parent,
+    URLClassLoader(@Nullable String name, URL[] urls, @Nullable ClassLoader parent,
```

Comment:
(Oops, I haven't intended to worry about package-private APIs, but I did here. No harm in it, at least.)